### ISSUE TITLE
(Voicing) Laundry list issue to check

### ISSUE THREAD

**Issue by terracoda** [phetsims/joist#705]
@jessegreenberg, I found these issues last week issue while reviewign GFLB design. I don't want to post individual issues as you may know about these issues issues already. Here are a check list that you can check off as you continue to work on teh Toolbar and the Preferences Menu to get them ready for dev testing:

- [x] With interactive highlights enabled, I see a pink interactive highlight at the edge of the sim before the Toolbar is enabled is enabled through voicing.
- [x] I see reading block highlights on reading blocks (mouse and keyboard) before Voicing is enabled.
- [x] Not clear where KB-focus is placed when Preference Menu is opened. I think it should go on the General Tab. With interactive highlights enabled and I open the Preferences dialog, this is what happens: On third tab stop I see edge of Toolbar button highlighted, on fourth tab stop a highlight appears on Close button, on fifth tab stop a highlight appears on the General tab.
- [x] On the Audio tab, when pressing Tab key, no buttons are focusable. The buttons in the Audio tab are not focusable with keyboard. The checkboxes, accordion, combobox, sliders are all focusable with keyboard. It’s just a problem with the on/off buttons.
- [x] No voicing in Preferences menu when Voicing is enabled.

Adding a couple more Toolbar issues after posting issue:
- [x] Voicing On/Off button not KB focusable - likely same issue as other on/off buttons.
- [x] Not sure where the Toolbar gets added into the Focus/Tab order. Is it at the top of the sim or just before Navbar?

**Comment by terracoda** [comment]
@jessegreenberg, if need details on any of these issues, or want me to make indy issues, just let me know.

**Comment by terracoda** [comment]
Adding my testing has been done on MacOS 10.13.6 in Safari.

**Comment by jessegreenberg** [comment]
These things are generally fixed. New issues can be made for any issues that are found during review.
